stm32-modules/thermocycler-refresh/firmware/system/STM32G491VETx_FLASH.ld

```diff
@@ -147,7 +162,25 @@ SECTIONS
     _edata = .;        /* define a global symbol at data end */
   } >RAM AT> FLASH
 
+  _siccmram = LOADADDR(.ccmram);
```

Comment:
are you planning on using this? i don't know if the startup asm here has support for that note below

Reply:
Nope... taking out this section, easy enough to add it back + update the assembly in if we want to use it later.